apps/www/next.config.js

```diff
@@ -25,12 +25,36 @@ const unprefixedStyleguideEnvVariables = {
     }, []),
 }
 
+function appendProtocol(href) {
+  if (href && !href.startsWith('http')) {
+    return `${NODE_ENV === 'production' ? 'https' : 'http'}://${href}`
+  }
+  return href
+}
+
+const PUBLIC_CDN_URL = appendProtocol(
```

Comment:
Also put this on `nextConfig.env`, so we can remove the fallback logic from lib/constants?